src/controllers/userController.js

```diff
@@ -14,8 +16,8 @@ const signUp = async (req, res) => {
     firstName, lastName, username, email, password
   } = req.body;
 
-  const foundEmail = await models.User.findOne({ where: { email } });
-  const foundUsername = await models.User.findOne({ where: { username } });
+  const foundEmail = await User.findOne({ where: { email } });
```

Comment:
is it possible to collapse these two queries into one?

Reply:
Yeah, it possible. Just seeing it from that angle now. I'll give that a go now. Thanks

Reply:
Hey @joel-ace , I tried to collapse the queries but it is not working as expected.
We have 2 scenarios I'm interested in handling with appropriate error messages respectively: 
1. a user with new email but an existing username
2. a user with a new username but an existing email

I could not find a way to query the DB with these two unique conditions, because the return object of one cannot be used to check the other.

I'll appreciate if you can point me in the right direction on how to solve this.

Reply:
I figured it out! Thanks.